spec/models/payment_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Payment, type: :model do
+  let!(:payment) { FactoryBot.build(:payment) }
+
+  it 'has a valid factory' do
```

Comment:
Create a special factory test that will check any factory's validity even some newly added

Basically there should be one test that check's all existing and newly added factories